### PR TITLE
mdn-bcd-collector v9.0.2 breakdown list (2023-04-24)

### PR DESCRIPTION
**Pull request by queengooborg**
This PR is a breakdown list of all the changes created by https://mdn-bcd-collector.gooborg.com v9.0.2 with the results collected as of April 24, 2023.  Results are limited to 2020+ browsers in this collector run.

Note that this PR is not intended to be merged directly.

### CODE REVIEW

File: javascript/builtins/Iterator.json [hunk 1]
```diff
@@ -48,18 +48,37 @@
             "spec_url": "https://tc39.es/ecma262/multipage/control-abstraction-objects.html#sec-%iteratorprototype%-@@iterator",
             "support": {
               "chrome": {
-                "version_added": "38"
+                "version_added": false
```
Comment (Josh-Cena) [inline javascript/builtins/Iterator.json:51]: `Iterator` is not a global (yet). You need to use `Symbol.iterator in Object.getPrototypeOf(Object.getPrototypeOf([].values()))`.

File: javascript/builtins/TypedArray.json [hunk 2]
```diff
@@ -1213,19 +1213,23 @@
             "spec_url": "https://tc39.es/ecma262/multipage/indexed-collections.html#sec-properties-of-the-typedarray-constructors",
```
Comment (Josh-Cena) [inline javascript/builtins/TypedArray.json:1213]: This page (`TypedArray.name`) has been deleted. we should delete this entry as well. (It's a static property that's exactly equivalent to function names.)

**Review by Josh-Cena — COMMENTED** [review]
I see some issues have been fixed; here are more suggestions

File: javascript/builtins/Intl/Segments.json [hunk 3]
```diff
@@ -87,13 +87,17 @@
               "spec_url": "https://tc39.es/ecma402/#sec-%segmentsprototype%-@@iterator",
               "support": {
                 "chrome": {
-                  "version_added": "87"
+                  "version_added": false
```
[the remark this answers is not in the document]
Comment (queengooborg) [reply javascript/builtins/Intl/Segments.json:90]: Already fixed these actually!  They'll be fixed in v9.0.3!

**Comment by queengooborg** [comment]
Superseded by #19486.

**Pull request closed without merging**